### ISSUE TITLE
StartDate week text color

### ISSUE THREAD

**Issue by its-felipe-almeida** [nomanr/WeekCalendar#20]
Is it possible to change the startDate week text color when it changes?

**Comment by com2ghz** [comment]
Check this pull: https://github.com/nomanr/WeekCalendar/pull/18

There is a OnWeekChangeListener now which is triggered when the week is changed.

**Comment by its-felipe-almeida** [comment]
thanks! is it on the same version (1.0.6)? I can't use here yet

**Comment by com2ghz** [comment]
Nope, not in the latest release yet. That nomanr bro needs to make a new release. But you can import this project as a module so you can use it. When there is a new release you switch back to the maven version.
